libs/ui-watt/src/lib/foundations/typography/+storybook/storybook-typography-overview.component.html

```diff
@@ -19,20 +19,49 @@ <h1>Typography</h1>
 </mat-card>
 
 <mat-card>
-  <h3>Font</h3>
-  <p>
-    <strong>Open Sans</strong> is the typeface used in the Watt Design System.
-  </p>
+  <h3>Font Family Options</h3>
+  <ul>
+    <li>Primary: <strong>Open Sans</strong></li>
+    <li>Secondary: <strong>Open Sans</strong></li>
+  </ul>
 </mat-card>
 
 <mat-card>
-  <h3>Type Scale</h3>
-  <p>
-    Font sizes are smaller on mobile and scale up at the md breakpoint to be
-    larger on desktop.
-  </p>
+  <h3>Font Weight Options</h3>
+  <ul>
+    <li class="font-weight-light">Light: 300</li>
+    <li class="font-weight-regular">Regular: 400</li>
+    <li class="font-weight-semi-bold">Semi-bold: 600</li>
+    <li class="font-weight-bold">Bold: 700</li>
+  </ul>
+</mat-card>
+
+<mat-card style="display: flex; gap: 64px">
+  <section>
+    <h3>Font Size Options</h3>
+    <p>size (line-height)</p>
+    <ul>
+      <li class="font-size-xxl">XXL: 32 px (42)</li>
+      <li class="font-size-xl">XL: 26 px (36)</li>
+      <li class="font-size-l">L: 20 px (28)</li>
+      <li class="font-size-m">M: 16 px (24)</li>
+      <li class="font-size-s">S: 14 px (20)</li>
+    </ul>
+  </section>
+  <section>
+    <h3>Letter Spacing Options</h3>
+    <ul>
+      <li>0</li>
+      <li>1,25</li>
+    </ul>
+  </section>
+</mat-card>
 
-  <table mat-table [dataSource]="dataSource" style="width: 100%">
+<mat-card>
+  <table mat-table [dataSource]="dataSourceLarge" style="width: 100%">
+    <caption>
+      <h3>LARGE SCREEN - Usage</h3>
```

Comment:
suggestion: Since we're using a mobile first approach, it makes more sense to start with default styles (i.e. small screen) and show the difference on large screen on the side. Sync with Kenneth if that would be acceptable.